cnmodel/data/_db.py

```diff
@@ -35,12 +35,12 @@ def _lookup(ind, *args, **kwds):
             data[k] = DATA[key][ind]
         return data
     else:
-            return DATA[key][ind]
+        return DATA[key][ind]
             
 def setval(val, *args, **kwds):
     key = mk_key(*args, **kwds)
-    if key in DATA:
-        raise RuntimeError("Data key '%s' has already been set." % str(key))
```

Comment:
Why remove this?

Reply:
If it is left in, we get an error:
File "/Users/pbmanis/Desktop/Python/cnmodel/cnmodel/data/_db.py", line 43, in setval
    raise RuntimeError("Data key '%s' has already been set." % str(key))

I am investigating this, will report back.

Reply:
Error in XM13_channels table header (duplicate header). Fixed table, will reinstate code as check for that error.